editor/filesystem_dock.cpp

```diff
@@ -1288,11 +1288,11 @@ void FileSystemDock::_make_scene_confirm() {
 }
 
 void FileSystemDock::_file_deleted(String p_file) {
-	emit_signal("file_deleted", p_file);
+	emit_signal("file_removed", p_file);
 }
 
 void FileSystemDock::_folder_deleted(String p_folder) {
-	emit_signal("folder_deleted", p_folder);
+	emit_signal("folder_removed", p_folder);
```

Comment:
These might have been better in the first commit that exposes FileSystemDock and could be cherry-picked easily, but I guess it's fine as is if you plan to make a matching 3.2 PR. No current code relies on those signals and the class wasn't exposed yet, so the bug couldn't be triggered.

Reply:
Might be worth renaming the methods too for consistency.

Reply:
As suggested, moved these changes to the first commit and renamed the methods as well.